src/Tribe/Views/V2/View.php

```diff
@@ -820,7 +839,14 @@ public function prev_url( $canonical = false, array $passthru_vars = [] ) {
 
 		if ( ! empty( $passthru_vars ) && ! empty( $url ) ) {
 			// Re-apply the pass-thru query arguments.
-			$url = add_query_arg( $passthru_vars, $url );
+			$query_args = array_merge( $query_args, $passthru_vars );
+		}
+
+		$query_args = $this->filter_query_args( $query_args, $url );
+		$query_args = array_filter( $query_args );
+
+		if ( ! empty( $url ) || ! empty( $query_args ) ) {
+			$url = add_query_arg( $query_args, $url );
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: d00a83257a6cd9b27ed4acefa8f932ef</sub>